curriculum/challenges/english/10-coding-interview-prep/rosetta-code/execute-a-markov-algorithm.md

```diff
@@ -123,7 +123,7 @@ let rules=[["A -> apple","B -> bag","S -> shop","T -> the","the shop -> my broth
             ["A -> apple","WWWW -> with","Bgage -> ->.*","B -> bag","->.* -> money","W -> WW","S -> .shop","T -> the","the shop -> my brother","a never used -> .terminating rule"],
             ["_+1 -> _1+","1+1 -> 11+","1! -> !1",",! -> !+","_! -> _","1*1 -> x,@y","1x -> xX","X, -> 1,1","X1 -> 1X","_x -> _X",",x -> ,X","y1 -> 1y","y_ -> _","1@1 -> x,@y","1@_ -> @_",",@_ -> !_","++ -> +","_1 -> 1","1+_ -> 1","_+_ -> "],
             ["A0 -> 1B","0A1 -> C01","1A1 -> C11","0B0 -> A01","1B0 -> A11","B1 -> 1B","0C0 -> B01","1C0 -> B11","0C1 -> H01","1C1 -> H11"]];
-let datas=["I bought a B of As from T S.",
+let data=["I bought a B of As from T S.",
```

Comment:
Looks like some indentation needs to be adjusted to keep the subsequent lines aligned.

Reply:
Fixed in https://github.com/freeCodeCamp/freeCodeCamp/pull/51246/commits/4d6f6ac2c79cb7cf674a5f911c204c1deef099a3